docs/dictionary/control_st/return.lcdoc

```diff
@@ -2,39 +2,142 @@ Name: return
 
 Type: control structure
 
-Syntax: return <value> 
+Syntax: return [ value | error ] <value>
 
-Summary: Stops the current <handler> and <return|returns> a <value> to the <handler> that <call|called> the current <handler>. Sets the result to the value as well
+Summary: Stops the current <handler> and <return|returns> a <value> to the
+<handler> that <call|called> the current <handler>.
 
 Introduced: 1.0
 
+Changed: 8.1.0
+
 OS: mac,windows,linux,ios,android
 
 Platforms: desktop,server,web,mobile
 
-Parameters:
-value: 
-handler: The name of the handler in which the return control structure appears.
+Example:
+    function simpleFunction
+       return 1
+    end simpleFunction
+    on testSimpleFunction
+       local tVar
+       put simpleFunction() into tVar
+       -- tVar contains 1, the result contains 1
+    end testSimpleFunction
+
+Example:
+    function simpleFunction
+       return value 1
+    end simpleFunction
+    on testSimpleFunction
+       local tVar
+       put simpleFunction() into tVar
+       -- tVar contains 1, the result contains empty
+    end testSimpleFunction
+
+Example:
+    function simpleFunction
+       return error 1
+    end simpleFunction
+    on testSimpleFunction
+       local tVar
+       put simpleFunction() into tVar
+       -- tVar contains empty, the result contains 1
+    end testSimpeFunction
+
+Example:
+    command simpleCommand
+       return 1
+    end simpleCommand
+    on testSimpleCommand
+       simpleCommand
+       -- the result contains 1
+    end testSimpleCommand
+
+Example:
+    function simpleCommand
+       return value 1
+    end simpleCommand
+    on testSimpleCommand
+       simpleCommand
```

Comment:
This is probably causing an issue, you might have to change to `get simpleCommand()`. Unless `simpleCommand` is meant to be a command.